guides/migrate-from-redshift.md

```diff
@@ -0,0 +1,190 @@
+1) Comment out the following by placing two ‘--’ in front of every occurrence of (or just delete it)
+  - ENCODE
+  - DISTSTYLE
+  - DISTKEY
+
+2) The ALTER statement tries to mark ownership to the user from AWS. We can remove that since the user for our purposes the user that creates the schema will be the owner anyway. Comment out “ALTER” same as above
+3) Comment or delete the three lines establishing the SORTKEY, that is also not used. Below is an example for just one of the tables after all the alterations are made. Everything commented out was from the original DDL downloaded from Redshift.
+ 
+* Example output after alterations
+    ```sql
+    --DROP TABLE tickit.users;
+    CREATE TABLE IF NOT EXISTS tickit.users
+    (
+        userid INTEGER NOT NULL  -- ENCODE RAW
+        ,username CHAR(8)   -- ENCODE lzo
+        ,firstname VARCHAR(30)   -- ENCODE lzo
+        ,lastname VARCHAR(30)   -- ENCODE lzo
+        ,city VARCHAR(30)   -- ENCODE lzo
+        ,state CHAR(2)   -- ENCODE lzo
+        ,email VARCHAR(100)   -- ENCODE lzo
+        ,phone CHAR(14)   -- ENCODE lzo
+        ,likesports BOOLEAN   -- ENCODE RAW
+        ,liketheatre BOOLEAN   -- ENCODE RAW
+        ,likeconcerts BOOLEAN   -- ENCODE RAW
+        ,likejazz BOOLEAN   -- ENCODE RAW
+        ,likeclassical BOOLEAN   -- ENCODE RAW
+        ,likeopera BOOLEAN   -- ENCODE RAW
+        ,likerock BOOLEAN   -- ENCODE RAW
+        ,likevegas BOOLEAN   -- ENCODE RAW
+        ,likebroadway BOOLEAN   -- ENCODE RAW
+        ,likemusicals BOOLEAN   -- ENCODE RAW
+    )
+
+    -- DISTSTYLE KEY
+     -- DISTKEY (userid)
+     -- SORTKEY (
+        -- userid
+        -- )
+    ;
+    -- ALTER TABLE tickit.users owner to "admin";
+
+    -- Permissions
+
+    -- GRANT RULE, SELECT, REFERENCES, TRIGGER ON TABLE tickit.users TO "admin";
+    -- GRANT SELECT ON TABLE tickit.users TO public;
+    ```
+
+
+### Create the database in Hydra
+1) Create a schema if you haven’t already done so
+2) Using that schema, run the modified DDL sql from the step above
+
+### Create the foreign tables to access the S3 parquet
+1) Access to S3 through parquet_s3_fdw
+
+    ```sql
+    CREATE EXTENSION parquet_s3_fdw;
+
+    CREATE SERVER parquet_s3_srv FOREIGN DATA WRAPPER parquet_s3_fdw OPTIONS (aws_region 'us-east-1');
+
+    CREATE USER MAPPING FOR CURRENT_USER SERVER parquet_s3_srv OPTIONS (user '/ACCESSKEY/', password '/SECRETKEY/');
+    Modify the CREATE TABLE statements for the foreign tables (example)
+    CREATE foreign TABLE
+    ```
+2) Simplify the data types
+  - All INTs and BOOLEAN to ‘int’
+     If you want to get tricky you can shorten or lengthen the ints to different sized ints, that won’t really do much except make more data available in memory cache at any given time. 
+     You will need to re-convert the 0-1 boolean ints back to boolean, we have a nice trick for doing that during the INSERT
+  - DATE and TIMESTAMP can be left alone
+  - All VARCHAR and other strings to simply ‘text’
```

Comment:
```suggestion
    - All INTs and BOOLEAN to `int`. If you wish, you can shorten or lengthen the ints to different sized ints which will affect memory usage. You will need to re-convert the 0-1 boolean ints back to boolean. This can be done during the INSERT.
    - DATE and TIMESTAMP can be left alone
    - All VARCHAR and other strings to simply `text`
```